### PR TITLE
Added myself to scoreboard and also added total

### PR DESCRIPTION
**Pull request by vsvipul555**
Added myself to scoreboard and also added total

### CODE REVIEW

File: _data/scores.yml [hunk 1]
```diff
@@ -362,7 +372,7 @@
     - name: Add more codechef solutions(8+3)(sent 2 PRs)
       link: https://github.com/faheemzunjani/Cool-Practice-Programs
       score: 20
-  total: 220
+    TOTAL: 220
```
Comment (gauthamzz) [inline _data/scores.yml]: why did you change this , please revert it back

Comment (vsvipul555) [inline _data/scores.yml]: oh my mistake
